README.md

```diff
@@ -1,32 +1,328 @@
-apimock
-=======
-[![Build Status](https://travis-ci.org/zezutom/apimock-1.svg?branch=develop)](https://travis-ci.org/zezutom/apimock-1)
+# ApiMock
 
-Data Mock proxy
+Do you depend on 3rd-party APIs? Does HTTP 503 "Service unavailable" sound way too familiar? There is absolutely
+no reason for why outages should slow you down.
 
+In essence, ApiMock is a HTTP proxy. It intercepts requests and replays captured responses. The tool is proven 
+in real-life projects. As such, it is easy to set up, has a minimal configuration with sensible defaults and comprises a set of thoughtful features. 
 
-Install:
-=======
+## Key Features
 
-run "npm run setup" on the folder where you checkout apimock
+* supports the most frequently used HTTP methods (GET, POST, PUT, DELETE)
+* caters for a variety of architectures - REST, SOAP, query parameters .. 
+* puts no restrictions on data formats - typically, but not only, JSON and XML
+* provides an authentic replay - status code and headers captured along with the data
+* enables to simulate live data - think stock quotes auto-updates
+* allows to organise and structure captured responses
 
+## Table of Contents
 
-Run:
-======
+- [Installation](#installation)
+- [Quick Start](#quick-start)
+- [Examples](#examples)
+  - [REST Style](#rest-style)
+  - [Query String](#query-string)
+TBC
 
-run "gulp"
+## Installation
 
+```git clone git@github.com:aalbuquerque73/apimock.git && cd apimock sudo npm install```
```

Comment:
sudo only needed if installed outside of the home folder, but I guess is good to have